satellite/satellitedb/migrate.go

```diff
@@ -380,54 +380,52 @@ func (db *DB) PostgresMigration() *migrate.Migration {
 						storage_node_id bytea NOT NULL,
 						PRIMARY KEY ( serial_number_id, storage_node_id )
 					)`,
-					`CREATE TABLE storagenode_bandwidth_rollups (
+					`CREATE TABLE storagenode_bandwidth_tallies (
```

Comment:
Existing migrations must not be changed unless we have made an explicit exception.

Reply:
this migration isn't in production yet, so we figured we would just modify the existing migration so that we don't have so many.

Reply:
Sure, then it's fine.